pyecobee/__init__.py

```diff
@@ -693,6 +693,43 @@ def set_aux_cutover_threshold(self, index: int, threshold: int) -> None:
         except (ExpiredTokenError, InvalidTokenError) as err:
             raise err
 
+    def update_climate_sensors(self, index: int, climate_name: str, sensor_names: list) -> None:
+        """Get current climate program."""
+        programs: dict = self.thermostats[index]["program"]
+        # Remove currentClimateRef key.
+        programs.pop("currentClimateRef", None)
+        program_to_edit = next(
+            climate for climate in programs["climates"] if climate["name"] == climate_name)
+
+        """Update climate sensors with sensor_names list."""
+        sensor_list = []
+        for name in sensor_names:
+            """Find the sensor id from the name."""
+            sensors = self.get_remote_sensors(index)
+            for sensor in sensors:
+                if sensor["name"] == name:
+                    sensor_list.append(
+                        {"id": "{}:1".format(sensor["id"]), "name": name})
+        program_to_edit["sensors"] = sensor_list
```

Comment:
`program_to_edit` is never referenced after this assignment.  Why is it here?

Reply:
This will show my ignorance of python, but since `program_to_edit` was reduced in line 701 from programs, the assignment also makes the change on `programs` variable. Though I could be way off, that is just how it seemed to work (pretty new to python).

Reply:
Hmm, I don't think that's right.  I would try and test with some debug statements to make sure it's doing what you expect.  The only change I see happening to `programs` is the pop on line 700.